shell/app/node_main.cc

```diff
@@ -146,6 +146,10 @@ int NodeMain(int argc, char* argv[]) {
     JavascriptEnvironment gin_env(loop);
 
     v8::Isolate* isolate = gin_env.isolate();
+    // TODO(ckerr) and/or TODO(codebytere) use node::SetIsolateMiscHandlers()
+    node::IsolateSettings is;
```

Comment:
for posterity's sake, can we note that this means `v8::MicrotasksPolicy::kExplicit`?

Reply:
I guess we could; but if that's the goal, we'd be better off just using `kExplicit` here instead of IsolateSettngs IMO

My rationale for using `node::IsolateSettings` wasn't so much "show that we're using kExplicit" as it was "show that we're using whatever Node is using" -- e.g. if they were to change from kExplicit to something else in their code, we would pick up that change automatically.